openlibrary/templates/site/body.html

```diff
@@ -29,12 +29,12 @@
       $for flash in get_flash_messages():
         <div class="$flash.type"><span>$flash.message</span></div>
       </div>
-      $# Paste next line into announcement variable to show the blue banner
-      $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = None
-      $if announcement:
-        <div class="page-banner page-banner-body">
-         $:announcement
-        </div>
+      $# Announcement banner will only be rendered if announcement and storage_key variables are set.
+      $# Be sure to escape any single quotes inside of the announcement HTML string.
+      $ announcement = ''
+      $# Using the current date in YYYY-MM-DD format for storage_key will prevent the use of duplicate keys.
+      $ storage_key = ''
+      $ announcement_bar = render_template('site/banner', announcement, storage_key) if announcement and storage_key else ''
+      $:announcement_bar
```

Comment:
```suggestion
      $if announcement and storage_key:
        $if ctx.user and ctx.user.preferences().get('hidden-banner') == storage_key:
          $render_template('site/banner', announcement, storage_key)
```
I wanted the banner rendering changes in `/templates/site/body.html`.  Please remove code that does the same from `/templates/site/banner.html` and add something like the above to `/site/body.html`.